packages/core/saved-objects/core-saved-objects-migration-server-internal/src/model/model.test.ts

```diff
@@ -2599,5 +2602,50 @@ describe('migrations v2 model', () => {
         expect(newState.retryDelay).toEqual(0);
       });
     });
+    describe('skipping reindex if not necessary', () => {
```

Comment:
nit we already have a pattern at the top of the file of:
```
describe('INIT', () => {
  describe('some general condition like skipping reindex if mappings the same', () => {
    it('INIT -> PREP...', () => {
       //
    }
  }
}
```

maybe we can move this test into that top-level `describe('INIT'`